### PR TITLE
Activator will be EOL-ed on May 24, 2017

### PR DESCRIPTION
**Pull request by eed3si9n**

### CODE REVIEW

**Comment by corruptmemory** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -1,154 +1,7 @@
-# Lightbend Activator
 
-[![Join the chat at https://gitter.im/typesafehub/activator](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/typesafehub/activator?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+**Important note: Activator will be EOL-ed on May 24, 2017.
+We recommend using `sbt new` command that integerates with Giter8 for general templating,
```
Comment (SethTisue) [inline README.md]: integrates

**Comment by corruptmemory** [comment]
@SethTisue you now have commit rights :-)

File: README.md (hunk 1 above)
Comment (eed3si9n) [inline README.md]: Fixed https://github.com/typesafehub/activator/commit/40d1bec9245d0bd2ad0da4fc4dabfc585a84038e

**Pull request merged**